### PR TITLE
SD-1450 As a Platform API application I can set public and private metadata for all resources

### PR DESCRIPTION
**Pull request by aleksandarpetrushev**
https://spark-solutions.atlassian.net/browse/SD-1450

### CODE REVIEW

**Comment by aleksandarpetrushev** [comment]
Need to add the metadata attributes for more resources, seems like I have left out some ones

File: api/spec/requests/spree/api/v2/platform/taxons_spec.rb [hunk 1]
```diff
@@ -80,4 +80,68 @@
       end
     end
   end
+
+  describe 'taxons#update for metadata' do
```
Comment (damianlegawiec) [inline api/spec/requests/spree/api/v2/platform/taxons_spec.rb:84]: @aleksandarpetrushev can we also create action?

Comment (aleksandarpetrushev) [reply api/spec/requests/spree/api/v2/platform/taxons_spec.rb:84]: yes

Comment (aleksandarpetrushev) [reply api/spec/requests/spree/api/v2/platform/taxons_spec.rb:84]: will try to refactor out shared examples

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**